openURL/src/comp.php

```diff
@@ -62,44 +60,57 @@
 		echo_message_to_screen(ERROR, "Could not open csv file - Process Stopped.");
 		exit;
     }
-	echo "Item ID" . "\t" . "Item Link" . "\t" . "Resource Link" . "\t" ."Item Title" . "\t" . "List Title" . "\t" . "Old Online Resource" . "\t" . "Old Online Link" . "\t" . "Outcome";
+	
+	// Write headersto audit file
+	$headers = [
+		"Item ID",
+		"Item Link", 
+		"Resource Link",
+		"Item Title",
+		"List Title",
+		"Old Online Resource",
+		"Old Online Link",
+		"Outcome"
+	];
+
+	echo "<table>";
+	writeToPage($headers);
+	writeToAuditFileOrExitOnFail($myfile, $headers);
+	
 	while (($line = fgetcsv($file_handle, 1000, ",")) !== FALSE) {
-		
 		$itemID = trim($line[0]);
-		fwrite($myfile, $itemID . "\t");
-		fwrite($myfile, "https://rl.talis.com/3/$shortCode/items/$itemID.html?lang=en-GB&login=1" . "\t");
-
-		$resourceData = getResource($shortCode, $itemID, $token, $TalisGUID);
-
-			$resourceID = $resourceData[0];
-			$itemTitle =  $resourceData[1];
-			$listTitle =  $resourceData[2];
-			$old_OnlineResource = $resourceData[3];
-			$old_OnlineLink = $resourceData[4];
-			echo "</br>";
-			echo $itemID . "\t";
-			fwrite($myfile, "https://$shortCode.rl.talis.com/resources/$resourceID.html" . "\t");
-			echo $itemTitle . "\t";
-			fwrite($myfile, $itemTitle . "\t");
-			echo $listTitle . "\t";
-			fwrite($myfile, $listTitle . "\t");
-			echo $old_OnlineResource . "\t";
-			fwrite($myfile, $old_OnlineResource . "\t");
-			echo $old_OnlineLink . "\t";
-			fwrite($myfile, $old_OnlineLink . "\t");
 
+		// Get resource Data
+		$resourceData = getResourceDataFromItemID($shortCode, $itemID, $token, $TalisGUID);
+		$resourceID = $resourceData[0];
+		$itemTitle =  $resourceData[1];
+		$listTitle =  $resourceData[2];
+		$old_OnlineResource = $resourceData[3];
+		$old_OnlineLink = $resourceData[4];
+
+		// Patch resource
 		$PatchOutcome = update_resource($shortCode, $token, $resourceID);
-			if ($PatchOutcome == 200) {
-				echo "Successfully updated to openURL";
-				fwrite($myfile, "Successfully updated to openURL\r\n");
-			} else {
-			echo "Not updated item - requires investigation.";
-			fwrite($myfile, "Not updated item - requires investigation\r\n");
-			}
+		$patchSuccessful = ($PatchOutcome == 200);
```

Comment:
Store a boolean true if the patch outcome is 200 (success).  This will be used later when choosing the audit outcome message.